webapp/src/modules/rental/selector.spec.ts

```diff
@@ -134,3 +136,41 @@ describe('when getting if the transaction is being submitted', () => {
     })
   })
 })
+
+describe("when getting rental status", () => {
+  describe("and there is a transaction submitting", () => {
+    beforeEach(() => {
+      rootState.rental.isSubmittingTransaction = true
+    })
+    
+    it('should return authorization status WAITING', () => {
```

Comment:
The word `an` is missing here, as we need to use an article:
```suggestion
    it('should return an authorization status WAITING', () => {
```